dateutil/tz/tz.py

```diff
@@ -1769,6 +1770,28 @@ def _datetime_to_timestamp(dt):
     return (dt.replace(tzinfo=None) - EPOCH).total_seconds()
 
 
+def supports_seconds():
```

Comment:
I would not like to change the public API at all with this, there should be no way to query this.

Also, this has a runtime cost when I would prefer an import-time cost.